app-header-layout/app-header-layout.html

```diff
@@ -147,6 +149,32 @@
         this._updateScroller();
       },
 
+      // Resizes app-header-layout to the height of the viewport and translates the content
+      // up to "simulate" disabled document scrolling on iOS.
+      simulateDisabledScroll: function(disabled) {
```

Comment:
what about if we expose a property named `disabled` following the the idea of a `disabled` state for `app-header